src/main/java/com/movie/store/model/CinemaHall.java

```diff
@@ -0,0 +1,33 @@
+package com.movie.store.model;
+
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@EqualsAndHashCode(of = {"capacity", "movieSessions"})
+@ToString(of = {"capacity"})
```

Comment:
it's make sense to have id in to string, that will make debug easier 
what about Data annotation, will it work here? same for other models?